packages/wallets/strike/src/adapter.ts

```diff
@@ -0,0 +1,346 @@
2zSyFFM258MRPZXysla7+qZvJX2gB1dRjHhDzBssZPLi5fetlp2caufY39YOW6t2pxbdzi0KLj5LjsmgehKSjZ5g+13kKUUe+8we297bx2TAhns+5uVvD/9RvGVvNs/ulL1n7Iwl4ZLkKYLD3wFnXuI7IVv1zWRXy5lxRKuqH1HAwC2yxi1Pf/KaNWI5DnF2FoNoMQAvv29OITn7gJO/cewLa3vxeLl/ZgulrFxNgQatUpqFSB0cC8iJREJrJmI67/k5DZR5QUMX6Nj1/DL6AfB+KfDipVfvDsuas2nBO8bsmBkSFS5yfAA2qSQ1vL54znJ7Jg5OBOm7Oay+9p6hNDp5Jx0aAQj22+acerrEmDazoZmxr4aqFh9FtbGOvk5TcEy59nhtypvcLGu3/QXLfvWtzG7s1EgkwYAwg32EFtRAgswcN5BpZLIUol83Umxqu4797t+PRNn8EZZ2/AO65/Lyiv4MjRIzh+5DDqU9NAphHwkgFBa6XjND5Wr8/8hRDJV6QU2a+rmv61aeGHs8dHh9PsO0tIV3hJdPWGZGZBcCI5DBbFsxpE3r2D5PUBF74ZyzN3E/6ZrQjmbmJKy/y02qTL+RFSQmeFb98ptObOBMTy+rQndVCa284EvhsJ63qu7WRPHlGCgPmN6SgfQWD+bWaMi/jQzwpoGJcItrWW0N5eQRAw1GbqeGrnbnz5ln/GI0/uxDv/8/uwZesVGBmuYv/+Axg4chLjQwOQaQJNDERayETUG9Pfm56ZuF4q9YQU6asyU/Gq4OJHsn3xaFp7MNZizyUdr924Pr18bv94k4zIA+CBzeA5n8X3JwU3EFfcIJQU3Dm59dXjNkmkLxiOIIWQbvyJuL/ftVIeu+58fhy335WhoYl/bbiFCi3STcNx+6l1/zAnO7DuJy4TDzjzYT2wG5ozUwbmjGLu3BZ0dbUgigKkSYpDB4/i1n/5Jr781Vux9rzN2HbN9QiDDjy3/zie3bMf/X1HMTLYjySegZBNCJnoWn1ysFod//tGs/7hRn1iTKlXz03+VR3n5vQssjY8d+G55U1/tbZt0/smsnrnjukfkqHWXWhfGKHS2gbOw3whKQVnHEEY+pKwM2bmnEMrBWYLH8XRJ1qQebmqGLUWtMzN1zmrOUJ9yDf5hP1zFuHOGEMQmtl96p6ihZPsnrGt5TLCKPCVy0q5BK0USuUIlVLkM/8oDNDRUUFHewXlcgSlFCbGpnDk8HE8+eSTeOjhh3DkxBDOWr8ZWjG0VNqgJUPSbKA6NYp6bRxCpciyGHHSrM7Ual+t16c/LTJ5cnL8yKsO4viNzPO38I10HTtn9dltWz60Iux+ewxUHqo+TI6yx8A762jraEGpUgHnke2Jh76n4O5q9+xxgxfUOoC4xhEPAr/A3N7Rzj7GFYEYC/KFL5gu5M9R5gEO3qbFkjrCIPAq4CDgKJdKVgtI/f3uQn1raxkd7RW0t5XQUonAA4ZGrYmR4TEcOHAIjz32BPbs2Y3evgNIU3OnG+qniRJhFCGKyiBEg3Gm47jZqNWnvteIsxszmT47NnDoN4bi+Y0CHSK+li8LetatCDa/56xg0zuobp8/yYfpgfKzmCn1IlaDCMoE5UobSuUywrBkwnAY2ZazJWPZBaSEWgVSnv27YhP3cMb8/zHnH8hNPd45eDIr33JPw9zQMX+Ock69oLQUBSiVSmaOnxBQalw7Qk5Rihi65raDEI0kTjE5OYmpqSqe3bcfz/X2YmB0EivXnIOBU8M4dvQg6tVxJMmMLYSFxi0kihCVW5UQcigT8rbq5PhX47TROzHaL/Ab/vitED3aWnpoq1yzoIt2vKOdnP2W06JzzkdXe6U5Z5o2K31o7aJo7WCQqgEQAakyVCfHkMkUWkujxwtLRhNoW6DOUp4xi2Kl1OcOxkCCW7mUSewCX5K1JM+Cz5CJHKZi556TYWAlb5SgXA7BqUaWNqFkirIVizJKIa1yqFqdxslTAzjUdwSnBgYwNDwKUIbW1jkABOozNSghoHVmm14KhGillG5oYGejUb9TQd1eb06NzUxO/daGZn+rSJf2OUsIQRgQ1bm0pOZfQfWiq1rCnkujSlt760LwhadXSMd8hlKFIE7rGB0ZxMjQccxMT0ArAa3N5ghCjjAqIQg5giBEGEVgjCOMyiiXS2A8RCkqzUoKaUF1zGwu4PBsSkloJYxfkUwhRIKQU+/Z0z7HTC01GnWkSYqAh6g3Gmg2Y3BeRnvnIghBcfhIH06eOIqJ8RMgMKIZxkNolfkOISVUZFk6nWXJzxrNxn1plmxXWp+s16czKdLfOmzr35Tp01JexKUM5gZBeSNFaR10aR2heh1jvJsHQQQtuNaamhMdgjBz4sNSiKhURs/qM3D66YugtcTAwBCOHn4WM9VxSJl5S3bK86aPmVySHrrsdHeOxmG6atLYtEURwjBCpaXVFLUIwXS1isnJCWSZRqnSgU0XXohypYKndz2NocF+JM0GhEgKlrhUSpFlUospJfVzIHR3s1F7Mk4aO+K4Md5oTGf4N/74nYI6hcEcFjtfe10AAAE5SURBVEWtQZJk7QRYG3C+HgSrtMZ8QthcAJ3Q6KKMzQnCcmnB4qWsvaOVAiDT1Sqt1WbI3M5ldPGSHnR2LkSlXEFrWxvKrSE4YyiVQxCqIWSK6ckp1KrjmBwfxvDISUs5SVGvTUIqgSWnrcE569dDyhi9+/fi5MnDnscXhS2gjINzpqUUilKmhMgUoIUU2RQI2SeyZE+SZnubcX035/xoksRJmsSi0az+TjERfuepXoyVKUAopZxwFlJiCgJMiKRFyawDoHMJ0XN5UOoMwrCL0bAzjFo6w7DcFUWVLkrRHoRhGAYRAxRjnFLOQsI5o1opOl0do3GzwSzUWTDGFSFE8TCUYRiqNEmEkkIqrQUBhFK6rpQYBjCklBjKRDYspBhWUo/MmdM2FsfN6WbSSLM0FdXqyO88P/H/A9Y7WPncwGZZAAAAAElFTkSuQmCC';
```

Comment:
Can you use an SVG here? This looks quite large.

Reply:
yup, sorry, this was just meant as a placeholder, we've updated it